docker-compose.yml

```diff
@@ -14,6 +14,22 @@ services:
       - dhis2:/opt/dhis2/
       #- ./config/dhis2/log4j2.xml:/opt/dhis2/log4j2.xml:ro
       - ./config/dhis2/dhis.conf:/opt/dhis2/dhis.conf:ro
+      - type: tmpfs
+        target: /tmp
+        tmpfs:
+          size: 1000000
```

Comment:
Would this be a bit clearer with e.g. "1m" instead of "1000000".

Reply:
I'm not sure if it's just intellij but it's showing as an error if I add units

Reply:
OK. Let's go with this for now then.